lambda_functions/crawl_bobae/lambda_function.py

```
@@ -0,0 +1,132 @@
+                dt = dt.replace("\xa0", " ")
+                dt = dt.split(" ")
+                dt = dt[0] + " " + dt[2]
+
+                data["date"] = int(datetime.strptime(dt, "%Y.%m.%d %H:%M").timestamp())
+
+                data["comments"] = []
+                comments = soup.find("div", "commentlistbox")
+                comments = comments.find("ul", "basiclist")
+                try:
+                    comments = comments.find_all("li")
+                except:
```

Ruff (0.8.2)

Do not use bare `except`

(E722)